gradio/__init__.py

```diff
@@ -24,6 +24,7 @@
     ClearButton,
     Code,
     ColorPicker,
+    Component,
```

Comment:
why?

Reply:
Sorry should have clarified, but in a downstream app that I'm creating, I needed a function that accepted any Gradio component. Right now, the type hint is `gr.components.Component` --> this lets it be `gr.Component`